docs/index.bs

```diff
@@ -3107,7 +3107,7 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
       1. Assert: |request|'s [=request/destination=] is not "<code>serviceworker</code>".
       1. If |request|'s [=request/destination=] is either "<code>embed</code>" or "<code>object</code>", then:
           1. Return null.
-      1. Else if |registration|'s <a>active worker</a>'s [=service worker/list of router rules=] is set:
+      1. Else if |registration|'s <a>active worker</a>'s [=service worker/list of router rules=] is [=list/is not empty=]:
```

Comment:
I think it is more elegant to remove this condition entirely, since if the list is empty, "Get Router Source" will return null, so the substep will not trigger.

Reply:
Since I do not think it is a good idea to ask the intern to work on this after the leave, let me take over the part.  I am going to merge this as-is, and will address that in the follow up PR, which might come next year.

Reply:
Anyway, thank you for your comment.

Reply:
It is totally OK for me to do such small things as an ooen source contributor, but thanks for your consideration.
Please feel free to ask me if there are any inconvenience due to my work!